101-aci-storage-file-share/azuredeploy.json

```diff
@@ -0,0 +1,104 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccountType": {
+            "type": "string",
+            "defaultValue": "Standard_LRS",
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_ZRS"
+            ],
+            "metadata": {
+                "description": "Storage Account type"
+            }
+        },
+        "storageAccountName": {
+            "type": "string",
+            "defaultValue": "[uniquestring(resourceGroup().id)]",
+            "metadata": {
+                "description": "Storage Account Name"
+            }
+        },
+        "fileShareName": {
+            "type": "string",
+            "defaultValue": "[uniquestring(resourceGroup().id)]",
+            "metadata": {
+                "description": "File Share Name"
+            }
+        },
+        "containerInstanceLocation": {
```

Comment:
if this is only available in certain locations let's add allowed values for now - if they are not available in westus (where CI runs) then add a valid defaultValue for location

Reply:
added, westus is supported.